src/app/rundown-view/components/mini-shelf/mini-shelf.component.ts

```diff
@@ -140,4 +146,10 @@ export class MiniShelfComponent implements OnInit, OnDestroy, OnChanges {
   public get mediaFilename(): string {
     return this.segment.metadata?.miniShelfVideoClipFile ?? ''
   }
+
+  public updatePositionInVideo(tooltipMetadata: TooltipMetadata): void {
+    const videoLengthInPixels: number = this.thumbnailRef.nativeElement.offsetWidth
+    const positionInVideoInPercent: number = Math.round((tooltipMetadata.horizontalOffsetInPixels / videoLengthInPixels) * 100)
+    this.positionInVideoInMs = Math.round((this.mediaDurationInMsWithoutPostroll / 100) * positionInVideoInPercent)
```

Comment:
In mathematics, percent is almost always expressed as floating point [0,1]. So the rounding and `* 100` can be removed, resulting in:
```typescript
    const positionInVideoInPercent: number = tooltipMetadata.horizontalOffsetInPixels / videoLengthInPixels
    this.positionInVideoInMs = Math.round(this.mediaDurationInMsWithoutPostroll * positionInVideoInPercent)
```

Reply:
Updated